.github/workflows/ci.yml

```diff
@@ -48,7 +48,6 @@ jobs:
       matrix:
         parameters:
           - "--skip_qc"
-          - "--remove_ribo_rna --skip_qualimap"
```

Comment:
Just to be sure as this is not strictly related to the "new software versions": This was removed on purpose?

Reply:
Yes, the CI tests for SortMERNA take forever to run and timeout all of the time now. I have given up on it doing anything purposeful now 😅

Reply:
I guess we could create a smaller database file to feed it for the tests 🤔 Let me have a look.

Reply:
Size of database files in the default [`--ribo_database_manifest`](https://github.com/nf-core/rnaseq/blob/master/assets/rrna-db-defaults.txt) we use with the pipeline. Will try to use only the smallest one for the purposes of the tests to see if it speeds things up.

```
735K    silva-arc-23s-id98.fasta
2.2M    rfam-5.8s-database-id98.fasta
3.8M    silva-arc-16s-id95.fasta
8.2M    rfam-5s-database-id98.fasta
13M     silva-euk-18s-id95.fasta
13M     silva-bac-23s-id98.fasta
15M     silva-euk-28s-id98.fasta
19M     silva-bac-16s-id90.fasta
```

Reply:
Will look at this in a follow up PR. Need to point to some code in this PR so will merge for now.